docs/cookbook/persistence/memory.md

```diff
@@ -0,0 +1,29 @@
+# In-Memory Wallet
+
+The simplest wallet is one that does _not_ have any persistence. All information about the wallet is held in memory, and will be destroyed upon termination of the process.
+
+In-memory implies that the addresses the wallet has revealed, the syncing that has been performed including data on the transaction graph will not persist when the wallet is destroyed, and related operations will need to be performed again.
+
+In general, this means performing a `full_sync()` when starting the wallet, because it has no knowledge of which addresses were given out and which scripts still have balances.
```

Comment:
```suggestion
In general, this means performing a `full_scan()` when starting the wallet, because it has no knowledge of which addresses were given out and which scripts still have balances.
```